modules/machine-vsphere-machines.adoc

```diff
@@ -0,0 +1,43 @@
+// Module included in the following assemblies:
+//
+// * machine_management/user_infra/adding-vsphere-compute-user-infra.adoc
+
+[id="machine-vsphere-machines_{context}"]
+= Creating more {op-system-first} machines in vSphere
+
+You can create more compute machines for your cluster that uses user-provisioned
+infrastructure on VMware vSphere.
+
+.Prerequisites
+
+* Obtain the base-64 encoded Ignition file for your compute machines.
+* You must have access to the vSphere template that you created for your cluster.
+
+.Procedure
+
+. After the template deploys, deploy a VM for a machine in the cluster.
+.. Right-click the template's name and click *Clone* -> *Clone to Virtual Machine*.
+.. On the *Select a name and folder* tab, specify a name for the VM. You might
+include the machine type in the name, such as `control-plane-0` or `compute-1`.
+.. On the *Select a name and folder* tab, select the name of the folder that
+you created for the cluster.
+.. On the *Select a compute resource* tab, select the name of a host in your
+datacenter.
+.. Optional: On the *Select storage* tab, customize the storage options.
+.. On the *Select clone options*, select
+*Customize this virtual machine's hardware*.
+.. On the *Customize hardware* tab, click *VM Options* -> *Advanced*.
+*** From the *Latency Sensitivity* list, select *High*.
+*** Click *Edit Configuration*, and on the *Configuration Parameters* window,
+click *Add Configuration Params*. Define the following parameter names and values:
+**** `guestinfo.ignition.config.data`: Paste the contents of the base64-encoded
+compute Ignition config file for this machine type.
+**** `guestinfo.ignition.config.data.encoding`: Specify `base64`.
```

Comment:
fyi i've noticed that the guestinfo params are not taken into account following the above steps, however in the next tab - `vApp_Properties` the params are exposed and so the OP only need to fill in the data value.

If you look in the ova template you will see the properties

```

Properties:
  Key:         guestinfo.ignition.config.data
  Label:       Ignition config data
  Type:        string
  Description: Inline Ignition config data

  Key:         guestinfo.ignition.config.data.encoding
  Label:       Ignition config data encoding
  Type:        string
  Description: Encoding for Ignition config data

```
@staebler fyi ^^